webpack/components/AnsibleHostDetail/AnsibleHostDetail.js

```diff
@@ -1,52 +1,45 @@
-import React, { useState } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import { Tabs, Tab, TabTitleText } from '@patternfly/react-core';
 import SkeletonLoader from 'foremanReact/components/common/SkeletonLoader';
-
 import { translate as __ } from 'foremanReact/common/I18n';
 
-import AnsibleVariableOverrides from './components/AnsibleVariableOverrides';
-
-import RolesTab from './components/RolesTab';
-import TabLayout from './components/TabLayout';
-
+import SecondaryRoutes from './components/SecondaryTabRoutes';
+import { ANSIBLE_KEY } from './constants';
 import './AnsibleHostDetail.scss';
-import WrappedAnsibleHostInventory from './components/AnsibleHostInventory';
 
-const AnsibleHostDetail = ({ response, status }) => {
-  // https://projects.theforeman.org/issues/32398
-  const [activeTab, setActiveTab] = useState('roles');
-  const handleTabClick = (event, tab) => setActiveTab(tab);
+const AnsibleHostDetail = ({
+  response: { id },
+  status,
+  location: { pathname },
+}) => {
+  const routeBuilder = subpath => `#/${ANSIBLE_KEY}/${subpath}`;
```

Comment:
nit: same function is in SecondaryTabRoutes, should we extract and reuse it?

Reply:
it's not exactly the same, but I extracted these to helper functions